images/base/Dockerfile

```diff
@@ -76,14 +76,16 @@ RUN curl -fsSL "https://download.docker.com/linux/$(. /etc/os-release; echo "$ID
     && clean-install "docker-ce=${DOCKER_VERSION}"
 
 # Enable CRI for containerd
-RUN mkdir -p /etc/docker && echo '{"cri-containerd": true}' > /etc/docker/daemon.json
-RUN mkdir -p /etc/containerd && containerd config default > /etc/containerd/config.toml
+RUN if [ -x "$(command -v containerd)" ]; then \
```

Comment:
does PPCLE not have recent enough packages? I'm also planning to upgrade the _Actual_ base image in that WIP containerd upgrade. (to 19.04)

Reply:
docker-ce 18.06.3 is the latest package we have so far in the docker website for power and we have opened and we have opened an issue with docker community to fix this issue - https://github.com/docker/for-linux/issues/629